generators/workspace/templates/plain/config/webpack.prod.js

```diff
@@ -0,0 +1,256 @@
+        usedExports: true, // should always be set to true
+        sideEffects: true, // should always be set to true as long as we don't change our code
+        portableRecords: false, // should always be set to false
+        flagIncludedChunks: true, // only in prod mode
+        occurrenceOrder: true, // only in prod mode
+        concatenateModules: true, // only in prod mode
+        moduleIds: 'hashed', 
+        chunkIds: 'total-size', // only in prod mode
+        runtimeChunk: 'single', //one runtime instance for all entries        
+        splitChunks: {
+            chunks: 'all',
+            minSize: 10000,
+            maxSize: workspaceMaxChunkSize,
+            cacheGroups: {
+                workspace: {
+                    test: workspaceRegex,
+                    priority: -5,
+                    name: 'vendor',
+                    enforce: true,
+                },
+                vendors: {
+                    test: /[\\/]node_modules[\\/]/,
+                    priority: -10,
+                    name: 'vendor',
+                    enforce: true,
+                }                
+
+            }
+        }
+    },    
+    plugins: [
+        new CleanWebpackPlugin({
+            cleanOnceBeforeBuildPatterns: [
+                '**/*',
+                path.join(process.cwd(), '../bundles/**/*')
+            ]
+        }),
+        new MiniCssExtractPlugin(),
+        ...HtmlWebpackPlugins,
+        new webpack.DefinePlugin({
+            'process.env.NODE_ENV': JSON.stringify(envMode),
+            'process.env.__VERSION__': JSON.stringify(appPkg.version),
+            'process.env.__LICENSE__': JSON.stringify(appPkg.license),
+            'process.env.__BUILD_ID__': JSON.stringify(buildId),
+            'process.env.__APP_CONTEXT__': JSON.stringify('/'),
+            'process.env.__DEBUG__': JSON.stringify(isDev)
+        }),
+        new CopyWebpackPlugin({
+            patterns: [
+                {
+                    from: workspaceMetaDataFile, to: base + '/bundles/phoveaMetaData.json',
+                    //generate meta data file
+                    transform(content, path) {
+                        return webpackHelper.generateMetaDataFile({buildId}, resolve(__dirname, '../' + defaultApp));
+                    }
+                },
+                //use package-lock json as buildInfo
+                {from: workspaceBuildInfoFile, to: base + '/bundles/buildInfo.json'}
+            ]
+        }),
+        //for debugging issues
+        /*new BundleAnalyzerPlugin({
+            // set to 'server' to start analyzer during build
+            analyzerMode: 'disabled',
+            generateStatsFile: true,
+            statsOptions: {source: false}
+        }),*/
+        new webpack.BannerPlugin({
+            banner: banner,
+            raw: true
+        })
+    ]
+};
+//console.log(JSON.stringify(config, null, '\t'));
+module.exports = config;
```

Comment:
Missing new line at the end of the file.

```suggestion
module.exports = config;

```